skate_tricks/database_operations.py

```diff
@@ -35,8 +33,8 @@ def post_new_trick(
     )
     db.add(db_new_trick)
     db.flush()
-    if fundamental_tricks is not None:
-        for fundamental_trick in fundamental_tricks:
+    if trick.fundamental_tricks is not None:
+        for fundamental_trick in trick.fundamental_tricks:
```

Comment:
We could improve this. Let's have a relationship table between tricks and fundamental_tricks, with a trick_id and a fundamental_trick_id. We can then submit fundamental tricks as integer ids rather than strings (in general, avoid strings for anything other than display text fields). We can then submit all relations in one query rather than looking them up indiidually. If we get a foreign key error, the caller fucked up!